LLAdd/LLAdd1.trig

```diff
@@ -973,4 +975,285 @@
+    owl:sameAs <http://example.com/LL/lagin.trig#CoirpreNiaFer>.
+
+<#Cathfad>
+    a foaf:Person;
+    irishRel:nomName "Cathfad";
+    foaf:title "drai"@sga, "druid"@eng;
+    rdfs:comment "IN mac aile nach airmighter a coitcinde";
+    rel:childOf <#RosaRuad>;
+    irishRel:numChild 1;
+    owl:sameAs <http://example.com/Laud_Misc_610/CGH/senchus_síl_hír.trig#Cathbad>.
+
+<#Conchabar>
+    a foaf:Person;
+    irishRel:nomName "Conchabar";
+    rel:childOf <#Nesa>, <#Cathfad>;
+    owl:sameAs <http://example.com/Laud_Misc_610/CGH/senchus_síl_hír.trig#Conchobor>.
+
+<#Nesa>
+    a foaf:Person;
+    foaf:gender "female";
+    irishRel:genName "Nesa";
+    owl:sameAs <http://example.com/Laud_Misc_610/CGH/senchus_síl_hír.trig#Nessa>.
+
+# p.379b33
+
+# Here follows a poem, 'Amra in mhaicni maicne rosa', which does not seem to add anything of
+# substance but might merit further examination. It can be found here:
+# https://celt.ucc.ie/published/G100051
+
+<#Mata-a6f46040>
+    a foaf:Person;
+    foaf:gender "female";
+    owl:sameAs <#Mata>;
+    irishRel:genName "Mata";
+    rdfs:comment ".i. Muirisge";
+    rel:childOf <#Mata-a6f46040>;
+    owl:sameAs <http://example.com/LL/lagin.trig#MataMurisc>.
+
+<#Cailitin>
+    a foaf:Person;
+    irishRel:genName "Cailitin";
+    irishRel:nomName "Eithne Cain";
+    irishRel:descendantOfGroup <#FeraibOllnecmaich>.
+
+<#FeraibOllnecmaich>
+    a irishRel:populationGroup;
+    irishRel:populationGroupName "Fir Ollnecmaich";
+    owl:sameAs <http://example.com/LL/lagin.trig#FeraibOoln-Écmacht>.
+
+<#Medhbh-a4f5b4a0>
+    a foaf:Person;
+    foaf:gender "female";
+    irishRel:nomName "Medhbh";
+    rel:childOf <#EchachFeidhlig>;
+    owl:sameAs <#Medhbh>;
+    rel:spouseOf <#Oilill-f41b3dc0>.
+
+<#EchachFeidhlig>
+    a foaf:Person;
+    irishRel:genName "Echach Feidhlig";
+    owl:sameAs <#EchachFeidlig>.
+
+<#Oilill-f41b3dc0>
+    a foaf:Person;
+    irishRel:nomName "Oilill";
+    owl:sameAs <#Oilill>.
+
+<#CoirpreCeinnderg>
+    a foaf:Person;
+    irishRel:nomName "Coirpre Ceinnderg";
+    owl:sameAs <#CoirpriCennderg>;
+    rel:siblingOf <#Medhbh-a4f5b4a0>.
+
+<#Oirlam>
+    a foaf:Person;
+    irishRel:nomName "Oirlam";
+    rel:childOf <#Medhbh-a4f5b4a0>, <#Oilill-f41b3dc0>.
+
+<#Cet>
+    a foaf:Person;
+    irishRel:nomName "Cet";
+    rel:childOf <#Medhbh-a4f5b4a0>, <#Oilill-f41b3dc0>.
+
+<#CainnerDerg>
+    a foaf:Person;
+    foaf:gender "female";
+    irishRel:nomName "Cainner Derg";
+    rel:childOf <#Medhbh-a4f5b4a0>, <#Oilill-f41b3dc0>.
+
+<#Sadb>
+    a foaf:Person;
+    foaf:gender "female";
+    irishRel:nomName "Sadb";
+    rel:childOf <#Medhbh-a4f5b4a0>, <#Oilill-f41b3dc0>.
+
+<#NaMained>
+    a foaf:Person;
+    irishRel:nomName "Na Maineda";
+    rel:childOf <#Medhbh-a4f5b4a0>, <#Oilill-f41b3dc0>.
+
+# <#NaMaineda> represents multiple people called Maine but, as the text doesn't
+# specify how many Ailill and Medb begat, a separate URI cannot be created for
+# each of them. EPT
```

Comment:
Yes, tricky.  I think we did something similar for another set.  I will have a think about how to deal with this.  This was the right call.

Reply:
Thanks!